package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "google_scraper_ruby",
+  "private": "true"
```

Comment:
I'm not sure why this file is likely empty here, normally this file contains some node package and script

Reply:
Found the error (spoiler alert: it was my bad). I didn't have the correct NodeJS, so the bootstrap process failed midway. I guess we can make the error more "eye-catching" (i.e. print it in red color). Should I create a PR to improve that?

Reply:
@Goose97 do you mean create a PR on the https://github.com/nimblehq/rails-templates? If yes, I would suggest to create an [issue](https://github.com/nimblehq/rails-templates/issues) for that first, and you can take it for sure (if you have time) :)